<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="danhSachMHP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnXoaTatCa">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>117</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Xoá tất cả</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnXoaMon">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>117</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Xoá môn</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
